### DISCUSSION TITLE
Account Suspension Issues - Request for Assistance

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132756] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Dear GitHub Support Team,

I hope this message finds you well. I am writing to express the difficulties I am facing following the suspension of my GitHub account(https://github.com/princemann56).

The suspension has significantly impacted my work and professional development. As a student and developer, I rely heavily on GitHub for managing and collaborating on key projects such as LearnFlow and Code-It. With my repositories now inaccessible, I am unable to push updates or maintain version control, which has stalled progress on these essential projects.

Additionally, my workflow has been severely disrupted. GitHub is an integral part of my daily routine, especially for code sharing and peer reviews. The suspension has also affected my ability to connect with potential employers and collaborators, as my GitHub profile serves as a critical portfolio to showcase my skills and contributions. Without access, I am unable to demonstrate my technical proficiency and project experience effectively.

This situation has created significant setbacks in both my academic and professional pursuits. I kindly request your assistance in resolving this matter at your earliest convenience, so I can continue my work without further hindrance.

Thank you for your understanding and support.

Sincerely,
Prince Mann

**Comment by ebndev** [comment]
Hi @princemann5656,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
